x-pack/plugin/ml/src/internalClusterTest/java/org/elasticsearch/xpack/ml/integration/JobResultsProviderIT.java

```diff
@@ -684,6 +686,9 @@ public void testGetSnapshots() {
     }
 
     public void testGetAutodetectParams() throws Exception {
+        PlainActionFuture<Boolean> future = new PlainActionFuture<>();
+        createStateIndexAndAliasIfNecessary(client(), ClusterState.EMPTY_STATE, new IndexNameExpressionResolver(), future);
+        future.get();
```

Comment:
I think you should move this into the `indexQuantiles` method, so it's closer to the reason why it's needed.  Without comments it's very hard to see why this test needs this and the others don't.